testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -906,6 +906,29 @@ protected static SortedSet<String> keywords(@Nullable String dialect) {
         .ok(expectedBigQuery);
   }
 
+  @Test void testDecimalLiteral() {
```

Comment:
I would remove the blank lines within this method. The test cases are so uniform that extra vertical space is not needed.

You should also test values that start and end with '.', values with no '.', values in scientific notation, the empty string, just '-', and invalid strings such as 'foo'. 

In BigQuery conformance you should allow values with double-quotes, e.g. `DECIMAL "-12.3"`.

Reply:
Oops, I meant dialect, not conformance. A couple of lines like this:
```
    final SqlParserFixture f = fixture().withDialect(BIG_QUERY);
    f.sql("select date '2020-10-10'")
        .ok("SELECT DATE '2020-10-10'");
```

Reply:
Hi @julianhyde , I get your point,I will improve the test content of this single test,thank you very much for your review.

Reply:
> I would remove the blank lines within this method. The test cases are so uniform that extra vertical space is not needed.
> 
> You should also test values that start and end with '.', values with no '.', values in scientific notation, the empty string, just '-', and invalid strings such as 'foo'.
> 
> In BigQuery conformance you should allow values with double-quotes, e.g. `DECIMAL "-12.3"`.

Hi @julianhyde ,I have modified the code according to your Review suggestion. If you have any ideas, please let me know. Thank you very much